pkg/virt-handler/vm.go

```diff
@@ -586,10 +587,12 @@ func (d *VirtualMachineController) updateVMIStatus(vmi *v1.VirtualMachineInstanc
 						if err != nil {
 							return err
 						}
-						if iface.PodIP != existingInterfaceStatusByName[domainInterface.Alias.Name].IP {
+
+						shouldCopy := !reflect.DeepEqual(existingInterfaceStatusByName[domainInterface.Alias.Name].IPs, iface.PodIPs)
```

Comment:
I think the variable is not needed, the expression can be used directly in the `if`.
Anyway, if you decide to keep the variable please change its name to something like -`shouldUpdatePodIps`.

Reply:
done